scripting/shavit-zones.sp

```diff
@@ -636,6 +654,83 @@ public Action Command_Modifier(int client, int args)
 	return Plugin_Handled;
 }
 
+//Krypt Custom Spawn Functions
+
```

Comment:
Please remove the newline after the comment, add a space after the double slashes and link to your GitHub profile